README change - preventing localhost URI for prod (#250)
SymfonyCasts/reset-password-bundle: README.md

    enable_garbage_collection: true
```

> Production environments may require the default_uri to be defined in config/packages/routing.yaml to prevent the URI in the email to point to localhost.

Reviewer: ```suggestion
The production environment may require the `default_uri` to be defined in the `config/packages/routing.yaml` to prevent the URI in emails to point to localhost.
```

```yaml

> when@prod:

Reviewer: ```suggestion
# config/packages/routing.yaml
when@prod:
```

    framework:
        router:

>             ...

Reviewer: ```suggestion
            # ...
```
